packages/cli/src/commands/build.ts

```diff
@@ -68,7 +70,14 @@ export const build: Command = {
 };
 
 async function run(options: BuildCommandOptions) {
-  const { watch, verbose, manifestFile, outputDir, clientConfig } = options;
+  const {
+    watch,
+    verbose,
+    manifestFile,
+    outputDir,
+    clientConfig,
+    codegen,
```

Comment:
does commander do this negation implicitly?

Reply:
Yeah https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue